### PR TITLE
Add strong type where it is possible

### PR DESCRIPTION
**Pull request by fezfez**
to merge after #32

### CODE REVIEW

**Comment by fezfez** [comment]
@Xerkus : since #32 is merged, this one is ready

**Comment by Xerkus** [comment]
@fezfez  can you rebase instead of merge?

**Comment by fezfez** [comment]
yep

File: src/PHPUnit/Controller/AbstractControllerTestCase.php [hunk 1]
```diff
@@ -194,30 +183,24 @@ public function getApplication()
 
     /**
      * Get the service manager of the application object
-     *
-     * @return ServiceManager
      */
-    public function getApplicationServiceLocator()
+    public function getApplicationServiceLocator(): ServiceManager
```
Comment (Xerkus) [inline src/PHPUnit/Controller/AbstractControllerTestCase.php]: Can this be typehinted with PSR container instead?

Comment (fezfez) [reply src/PHPUnit/Controller/AbstractControllerTestCase.php]: done

Comment (froschdesign) [reply src/PHPUnit/Controller/AbstractControllerTestCase.php]: It should the `Psr\Container\ContainerInterface` and not any interface from Laminas or laminas-servicemanager.

Comment (fezfez) [reply src/PHPUnit/Controller/AbstractControllerTestCase.php]: done

Comment (fezfez) [reply src/PHPUnit/Controller/AbstractControllerTestCase.php]: @froschdesign : this change make the build fail, can you check this ?

**Comment by Ocramius** [comment]
@fezfez closed on purpose?

**Pull request closed without merging**